vision/google/cloud/vision/geometry.py

```diff
@@ -219,7 +231,8 @@ def x_coordinate(self):
         :rtype: float
         :returns: X position coordinate.
         """
-        return self._x_coordinate
+        if self._x_coordinate != 0:
+            return self._x_coordinate
```

Comment:
No need for this branch if you don't store `0` in the constructor (i.e. just use `None`)